www/_posts/2017-11-10-plugins-release.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: post
+author:
+    name: Steve Gill
+    url: https://twitter.com/stevesgill
+title:  "Plugins Release"
+categories: news
+tags: release plugins
+---
+
+The following plugins were updated today because they are being deprecated:
```

Comment:
This makes it sound like all of these plugins are no longer supported, which I think is not the case?

Reply:
+1, sounds like all of them are being deprecated

Reply:
hmm i think that was a copy paste mistake. Good catch